spec/system/task_spec.rb

```diff
@@ -85,8 +78,7 @@
 
       it '既にステータスが完了のタスクのステータスを変更した場合、Taskの完了日が更新されないこと' do
         # TODO: FactoryBotのtraitを利用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id, status: :done, completion_date: Time.current.yesterday)
+        task = create(:task, :done)
```

Comment:
変数名を変えておいた方がデータがどのような内容かが伝わりやすいです。
```suggestion
        done_task = create(:task, :done)
```

Reply:
abf599a048cd375b28a3cf2d01df4e4e08ed9b5c　変数名を変更しました